refs 買いコメント、不安コメント一覧ページへの導線を作成

mterada1228 opened the pull request.

mterada1228 committed:
refs 買いコメント、不安コメント一覧ページへの導線を作成

ershn commented on app/controllers/races/horses/positive_comments_controller.rb:
```diff
@@ -0,0 +1,9 @@
+      end
+    end
+  end
+end
```
コメントモデルが一緒であれば、コントローラーも一緒にしてもいいと思いました。

どのコメントを表示するかはパラメーターなどで制御できますので。

ershn replied:
OK

mterada1228 committed:
指摘対応
  - positive_comments_controller と negative_comments_controller は
    comments_controller にまとめる

ershn commented on app/controllers/races/horses/comments_controller.rb:
```diff
@@ -0,0 +1,10 @@
+    class CommentsController < ApplicationController
+      def index
+        # TODO: 処理を実装
+        @position = params[:position]
```
意見の種類を表すためでしたら、以下のような名前の方が適切だと思います。
```
type=positive
type=negative
```

mterada1228 committed:
指摘対応
  - position => type に修正
